### PR TITLE
Simple quickstart

### PR DESCRIPTION
**Pull request by uF4No**
# What :computer: 
* Splits quickstart into two separate tutorials
* Updates to use zksync-ethers v6 and Vue.js v3

# Why :hand:
- We want a 5 minutes quickstart, simpler.
- Frontend updated to use our latests packages so it's not outdated

# Evidence :camera:
In Preview

# Notes :memo:
- Banner could be improved.
- **This PR needs to me merged at the same time as [this PR in the tutorials code repo](https://github.com/matter-labs/tutorials/pull/33)**

### CODE REVIEW

File: docs/assets/images/cli-doc-6.gif
Comment (MexicanAce) [inline docs/assets/images/cli-doc-6.gif:1]: I like the idea of a Banner image, but the GIF might be too much/distracting... Also unsure if we want to have hard-coded English on the site?

File: docs/build/quick-start/hello-world.md [hunk 1]
```diff
@@ -5,76 +5,70 @@ head:
...
-WALLET_PRIVATE_KEY=YourPrivateKeyHere...
+```bash
+? What type of project do you want to create? Contracts
+? Template: Hardhat + Solidity
+? Private key of the wallet responsible for deploying contracts (optional) ***************************************************
+? Package manager: yarn
 ```
 
-Your private key will be used for paying the costs of deploying the smart contract.
+The private key of your wallet will be included in the `.env` file of the project and won't be pushed to GitHub.
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:59]: Should we wrap this in a Warning/Info?

File: docs/build/quick-start/hello-world.md [hunk 2]
```diff
@@ -5,76 +5,70 @@ head:
...
 
-### Compile and deploy the Greeter contract
+The project structure is pretty straight forward:
 
-We store all the smart contracts' `*.sol` files in the `contracts` folder. The `deploy` folder contains all scripts related to deployments.
+- `hardhat.config.ts` contains the general configuration for Hardhat and the zkSync plugins, which are already imported and setup.
+- `/contracts` contains smart contracts. zkSync-CLI provides common examples like an ERC20, an NFT, and the Greeter contract that we'll use later on.
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:64]: We should use consistent capitalization of the `zksync-cli` tool
```suggestion
- `/contracts` contains smart contracts. `zksync-cli` provides common examples like an ERC20, an NFT, and the Greeter contract that we'll use later on.
```

File: docs/build/quick-start/hello-world.md [hunk 3]
```diff
@@ -93,629 +87,206 @@ contract Greeter {
 }
 ```
 
-2. Compile the contract with the following command:
+As you can see, it a simple Solidity contract with two methods to read a message, `greet()`, and modify it, `setGreeting()`.
 
-```sh
-yarn hardhat compile
-```
+::: tip Takeway
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:92]: typo
```suggestion
::: tip Takeaway
```

File: docs/build/quick-start/hello-world.md [hunk 4]
```diff
@@ -93,629 +87,206 @@ contract Greeter {
...
-```typescript
-export const deployContract = async (contractArtifactName: string, constructorArguments?: any[], options?: DeployContractOptions) => {
-  const log = (message: string) => {
-    if (!options?.silent) console.log(message);
-  };
+- `zksolc` for Solidity contracts.
+- `zkvyper` for Vyper contracts.
 
-  log(`\nStarting deployment process of "${contractArtifactName}"...`);
+As this is a Solidity project, it already has the `hardhat-zksync-solc` plugin installed and configured so there's nothing you need to setup. To compile the contracts in the project run the following command:
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:105]: nit: Add missing comma
```suggestion
As this is a Solidity project, it already has the `hardhat-zksync-solc` plugin installed and configured so there's nothing you need to setup. To compile the contracts in the project, run the following command:
```

File: docs/build/quick-start/hello-world.md [hunk 5]
```diff
@@ -93,629 +87,206 @@ contract Greeter {
...
-
-#### Externally owned accounts (EOAs)
-
-In order to interact with dApps built on zkSync, connect the MetaMask wallet to the zkSync Era Testnet.
-
-- Follow [this guide](./add-zksync-to-metamask.md) to connect Metamask to zkSync.
+The compiled artifacts will be located in the `/artifacts-zk` folder. These artifacts are similar to the ones generated by the Solidity compiler. For example, the ABI of the Greeter contract will be located in `/artifacts-zk/contracts/Greeter.sol/Greeter.json`.
 
-Please note, that login functionality for "Hello, world" will be implemented in the next steps.
+::: tip Takeway
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:134]: typo:
```suggestion
::: tip Takeaway
```

File: docs/build/quick-start/hello-world.md [hunk 6]
```diff
@@ -93,629 +87,206 @@ contract Greeter {
...
-},
-async getGreeting() {
-    return await this.contract.greet();
-},
+Requesting contract verification...
+Your verification ID is: 1781
 ```
 
-After connecting the Metamask wallet to zkSync Era Testnet, you should see the following page:
+Congratulations! You just deployed a smart contract to zkSync Sepolia testnet. You can find it in the [zkSync Sepolia explorer](https://sepolia.explorer.zksync.io/) by searching the contract address.
```
Comment (MexicanAce) [inline docs/build/quick-start/hello-world.md:192]: Suggestion, to bring more visibility/cheer, should we include an emoji at the beginning?
```suggestion
🥳 Congratulations! You just deployed a smart contract to zkSync Sepolia testnet. You can find it in the [zkSync Sepolia explorer](https://sepolia.explorer.zksync.io/) by searching the contract address.
```

File: docs/assets/images/cli-doc-6.gif
Comment (MexicanAce) [reply docs/assets/images/cli-doc-6.gif:1]: Defer for now

**Review by MexicanAce — DISMISSED** [review]
Overall, I like the content and flow of instructions. I think there's another iteration where we provide more visuals, but this is still way better than what we had before 🙌

**Review by MexicanAce — APPROVED** [review]

**Review by Romsters — APPROVED** [review]

**Review by Romsters — APPROVED** [review]

**Pull request merged**